packages/expo-updates/android/src/main/java/expo/modules/updates/errorrecovery/ErrorRecoveryHandler.kt

```diff
@@ -37,7 +40,10 @@ internal class ErrorRecoveryHandler(
 
   override fun handleMessage(msg: Message) {
     when (msg.what) {
-      MessageType.EXCEPTION_ENCOUNTERED -> maybeStartPipeline(msg.obj as Exception)
+      MessageType.EXCEPTION_ENCOUNTERED -> {
+        logger.error("ErrorRecoveryHandler: Exception encountered: ${(msg.obj as Exception).localizedMessage}", UpdatesErrorCode.Unknown, msg.obj as Exception)
```

Comment:
I think this will probably just end up being a duplicate of https://github.com/expo/expo/pull/18810/files#diff-e52efeb214dc970084cc6f093b1102d318451cd43f31f9d15dd795f720b47530R45, I would remove one of them

Reply:
Removed